src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_read_obs.F90

```diff
@@ -7657,6 +8165,199 @@ subroutine scale_obs_tskin_zscore( N_catd, tile_coord,   &
     
   end subroutine scale_obs_tskin_zscore
 
+ 
+  ! *****************************************************************
+  
+  subroutine scale_obs_sfmc_zscore( N_catd, tile_coord,         &
+       date_time, this_obs_param, tmp_lon, tmp_lat, tmp_time,   &
+       tmp_obs, tmp_std_obs )
+    
+    ! scale sfmc obs to model climatology via standard-normal-deviate (zscore)
+    ! scaling using seasonally varying (pentad) stats
+    ! 
+    ! scaling parameters are on 0.25 deg lat/lon grid
+    !
+    ! intended for scaling of ASCAT "sfds" observations 
+    !
+    ! A M Fox, reichle, April 2023
+    
+    use netcdf   
+    implicit none
+    
+    integer, intent(in) :: N_catd
+    
+    type(tile_coord_type), dimension(:), pointer :: tile_coord    ! input
+    
+    type(date_time_type), intent(in) :: date_time
+    
+    type(obs_param_type), intent(in) :: this_obs_param
+
+    real,    intent(in),    dimension(N_catd) :: tmp_lon, tmp_lat
+    real*8,  intent(in),    dimension(N_catd) :: tmp_time         ! J2000 seconds
+        
+    ! inout
+    
+    real,    intent(inout), dimension(N_catd) :: tmp_obs
+    real,    intent(inout), dimension(N_catd) :: tmp_std_obs
+    
+    ! ----------------------------------------------------------
+    ! Grid and netcdf parameters (might want to read these from netCDF file?)
```

Comment:
Yes, I think that's better. Or, at the very least, we need to check that the N_lat and N_lon that are read from the file are consistent with the hardcoded dlat and dlon (and - if that can be done sensibly - also ll_lat and ll_lon).